src/server/getters/get-markets-info.ts

```diff
@@ -23,15 +21,16 @@ export function getMarketsInfo(db: Knex, universe: Address|null|undefined, marke
     query = `${query} "marketID" IN (??)`;
     queryParams = [marketIDs!];
   }
+  query = `${query} ORDER BY "creationTime" ASC`;
```

Comment:
Not sure what the goal is in sorting this here. @stephensprinkle  do you know what order is desired for querying multiple markets at a time (like those that belong to a universe?)